### ISSUE TITLE
Allow users to inject their own sector_classification

### ISSUE THREAD

**Issue by maurolepore** [2DegreesInvesting/r2dii.match#353]
Relates to https://github.com/2DegreesInvesting/r2dii.data/issues/199#issuecomment-780090242

**Comment by jdhoffa** [comment]
Just my thoughts, I can see this easily being done by allowing for the "special" sector_classification input "MANUAL". ie. a loanbook might look something like: 
``` r
tibble::tribble(
  ~sector_classification_system, ~id_ultimate_parent,             ~name_ultimate_parent, ~id_direct_loantaker,                ~name_direct_loantaker, ~sector_classification_direct_loantaker,
  "MANUAL",              "UP15", "Alpine Knits India Pvt. Limited",               "C294", "Yuamen Xinneng Thermal Power Co Ltd",                                 "power"
)
#> # A tibble: 1 x 6
#>   sector_classifi… id_ultimate_par… name_ultimate_p… id_direct_loant…
#>   <chr>            <chr>            <chr>            <chr>           
#> 1 MANUAL           UP15             Alpine Knits In… C294            
#> # … with 2 more variables: name_direct_loantaker <chr>,
#> #   sector_classification_direct_loantaker <chr>
```

<sup>Created on 2021-02-19 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

The column `sector_classification_system` would have to have the entry `MANUAL`, and the column `sector_classification_direct_loantaker`, would have to be one of the PACTA sectors (as specified, for example, in `r2dii.data::green_or_brown`). 

When such an entry is passed, we could bypass the sector classification join, and just take the input value. 

Just a thought!

**Comment by maurolepore** [comment]
Do we need a blog post or new article to show users how to use "MANUAL"?

**Comment by jdhoffa** [comment]
So as far as I know, it won't actually work with "MANUAL" just yet? we still need to code the solution. This is just a proposition

**Comment by maurolepore** [comment]
Oh, thanks for explaining.

And what you think about this solution?: https://github.com/2DegreesInvesting/r2dii.match/pull/354

I'm not trying to push for it. I nudge you to it mostly because it's already implemented. If the "MANUAL" approach wins, I'm happy to close #354 to avoid confusion.

**Comment by maurolepore** [comment]
It seems we implemented #354 and released it in 0.0.9

https://2degreesinvesting.github.io/r2dii.match/news/index.html#r2dii-match-0-0-9-2021-03-10
